R/parallel.R

```diff
@@ -67,29 +67,42 @@ hipercow_parallel_get_cores <- function() {
 
 ##' Sets the environment variables `MC_CORES`, `OMP_NUM_THREADS`,
 ##' `OMP_THREAD_LIMIT`, `R_DATATABLE_NUM_THREADS` and `HIPERCOW_CORES` to
-##' the given number of cores. This is used to ensure that parallel workers you
-##' launch using [hipercow_parallel] will have the correct resources, but you
+##' the given number of cores. This is used to help various thread-capable
+##' packages use the correct number of cores. You
 ##' can also call it yourself if you know specifically how many cores you want
 ##' to be available to code that looks up these environment variables.
 ##'
 ##' @title Set various environment variables that report the number of cores
 ##' available for execution.
 ##'
+##' @inheritParams task_eval
```

Comment:
Need a correct doc entry here for `@envir` ; the one from task_eval is not correct!

Here it's to scope the lifetime of the environment variable changes, perhaps with a hint to the user that they leave it alone or consult `withr::local_envvar` for details

Reply:
Ok - I think that's better now?